_spinnaker_install_admin_guides/install-on-aws-halyard-pod.md

```diff
@@ -0,0 +1,668 @@
+5. Click on "Add inline policy" (on the right)
+6. Click on the "JSON" tab
+7. Add this text (replace `s3-spinnaker-abcxyz` with the name of your bucket)
+
+	```json
+	{
+	  "Version": "2012-10-17",
+	  "Statement": [
+	    {
+	      "Effect": "Allow",
+	      "Action": "s3:*",
+	      "Resource": [
+	        "arn:aws:s3:::spinnaker-abcxyz",
+	        "arn:aws:s3:::spinnaker-abcxyz/*"
+	      ]
+	    }
+	  ]
+	}
+	```
+
+8. Click on "Review Policy"
+
+9. Give your inline policy some name. For example `s3-spinnaker-abcxyz`
+
+10. Click "Create Policy"
+
+# Start the Halyard container
+
+Halyard is a tool used to install Spinnaker (it generates Kubernetes manifests for each of the Spinnaker services).  First, create a namespace for Spinnaker to run in:
+
+```bash
+# Feel free to use a different namespace for this
+kubectl create ns spinnaker
+```
+You can use this manifest to install and run Halyard (call the file `halyard.yml`)
+
+```yml
+# halyard.yml
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: spinnaker
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- 
+  kind: ServiceAccount
+  name: default
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: hal-pvc
+  labels:
+    app: halyard
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 100Mi
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: halyard
+spec:
+  replicas: 1
+  serviceName: halyard
+  selector:
+    matchLabels:
+      app: halyard
+  template:
+    metadata:
+      labels:
+        app: halyard
+    spec:
+      containers:
+      - 
+        name: halyard
+        image: armory/halyard-armory:{{ site.data.versions.halyard-armory-version }}
+        volumeMounts:
+        - 
+          name: hal
+          mountPath: /home/spinnaker/.hal
+        env:
+        - 
+          name: HOME
+          value: "/home/spinnaker"
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 65535
+        fsGroup: 65535
+      volumes:
+      - 
+        name: hal
+        persistentVolumeClaim:
+          claimName: hal-pvc
+```
+
+Apply (create) the resources:
+
+```bash
+kubectl -n spinnaker apply -f halyard.yml
+```
+
+Check the status of the pod, until it's up (and troubleshoot accordingly, based on standard Kubernetes troubleshooting):
+
+```bash
+kubectl -n spinnaker get pods
+```
+
+# Enter the Halyard container
+
+```bash
+kubectl -n spinnaker exec -it halyard-0 bash
+
+# Also, once in the container, you can run these commands for a friendlier environment to:
+# - prompt with information
+# - alias for ls
+# - cd to the home directory
+export PS1="\h:\w \u\$ "
+alias ll='ls -alh'
+cd ~
+```
+
+# Configure Spinnaker to install in Kubernetes
+
+Inside the container, use the Halyard `hal` command line tool to add a Kubernetes account using your minified kubeconfig
```

Comment:
i don't recall reading about minified kubeconfig earlier (so probably just nix `minified`

Reply:
actually, if i'm reading this correctly, we can delete lines 330 ~ 332 ... since we're not using kubeconfigs to for new providers.